Update dump-restore docs (#11743)
ydb-platform/ydb: ydb/docs/en/core/reference/ydb-cli/export-import/_includes/tools_dump.md

`--save-partial-result`: Don't delete the result of partial dumping. Without this option, the dumps that terminated with an error are deleted.

> `--preserve-pool-kinds`: Preserve storage pool kind settings. If this option is enabled, storage pool kind will be saved to dump. In this case, if there will be no such storage pool kind in database on restore, error will occur. By default this option is disabled and any existing storage pool kind will be used on restore.

Reviewer: ```suggestion
`--preserve-pool-kinds`: If this option is enabled, the `tools dump` command saves storage pool names to the dump. To import such a dump, the names of storage pools in the database must match the names saved in the dump. If at least one storage pool with the required name is missing, the import procedure will end with an error. By default this option is disabled, and the import procedure will use any existing storage pools regardless of their names.
```

Author: I suggest it's necessary to use 'storage pool kind' instead of 'storage pool name', because it's the way they're referenced in the configuration: https://ydb.tech/docs/en/reference/configuration/#syntax3 

changed the rest of the description as suggested though

> `--preserve-pool-kinds`: If this option is enabled, `tools dump` command saves storage pool kind settings to the dump. To import such a dump, same storage pools kinds must be present in the database. If at least one is missing, the import procedure will end with an error. By default this option is disabled, and the import procedure will use any existing storage pool kinds.

Reviewer: ```suggestion
`--preserve-pool-kinds`: If this option is enabled, the `tools dump` command saves information about the usage of storage pools by each table to the dump. To import such a dump, the same storage pools must be present in the database. If at least one is missing, the import procedure will end with an error. By default this option is disabled, and the import procedure will use any existing storage pools.
```

Reviewer 2: 1. Let's add a link to https://ydb.tech/docs/en/concepts/glossary#storage-pool
2. What does it actually save? Just the storage pool name or some extra settings? Let's be as specific as possible.
3. What does "any existing storage pools" mean? If there are multiple configured pools, would one be chosen at random? I thought there should be some default one.

Author: 1. Done
2. It actually saves the value of `DATA` parameter in `FAMILY` section of `CREATE TABLE` statement https://ydb.tech/docs/en/yql/reference/syntax/create_table/family
3. So if `--preserve-pool-kinds` is not used, this setting for column families will not be saved and column families will use the default storage pool kind specified at the moment of database creation: https://ydb.tech/docs/en/yql/reference/syntax/create_table/family

Changed the description accordingly.
